gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/AbstractJdbcRepository.java

```diff
@@ -37,13 +39,17 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static org.springframework.data.relational.core.query.Criteria.where;
+
 /**
  * @author Eric LELEU (eric.leleu at graviteesource.com)
  * @author GraviteeSource Team
  */
 public abstract class AbstractJdbcRepository {
+    public static final String USER_ID_FIELD = "user_id";
+    public static final String USER_EXTERNAL_ID_FIELD = "user_external_id";
+    public static final String USER_SOURCE_FIELD = "user_source";
```

Comment:
```suggestion
    private static final String USER_ID_FIELD = "user_id";
    private static final String USER_EXTERNAL_ID_FIELD = "user_external_id";
    private static final String USER_SOURCE_FIELD = "user_source";
```

Reply:
with another package structure these could be package-private, but I think they should be _at least_ protected - so that concrete repo implementation can reuse them. If they're left public that allows using these for `@Column` annotations for these fields, which might be good for keeping consistency beetween different entities sharing fields.